extra-functions2/extra-07/script.js

```diff
@@ -0,0 +1,8 @@
+function purchase(inventary, item) {
+  if (inventary.includes(item)) {
+    return "Hola";
```

Comment:
this message does not make sense 🤔 

Should say something along the line of "the item is included in the inventary"